imports/ui/views/Auth/ResetPassword.vue

```diff
@@ -1,35 +1,37 @@
 <template>
   <div>
     <div class="title">Reset password</div>
-      <Form as="v-form" @submit.prevent="resetPassword" ref="setPasswordFormObserver">
-        <Field v-slot="{errors}" name="new password" vid="confirmation"
-                            rules="strength_password|required">
-          <v-text-field v-model="user.password" id="inputNewPassword"
+    <Form as="v-form" v-slot="{handleSubmit}" ref="setPasswordFormObserver">
+      <v-form @submit="handleSubmit($event, resetPassword)" autocomplete="off">
+        <Field v-slot="{field, errors}" name="password" rules="strength_password|required">
```

Comment:
verify the functionality for this since I remembered I had problems with the initial values for this form when I added the field slot.